crates/nu-command/src/experimental/job_recv.rs

```diff
@@ -32,7 +32,7 @@ Messages may have numeric flags attached to them. This commands supports filteri
 If no tag is specified, this command will accept any message.
 
 If no message with the specified tag (if any) is available in the mailbox, this command will block the current thread until one arrives.
-By default this command block indefinitely until a matching message arrives, but a timeout duration can be specified. 
+By default this command block indefinitely until a matching message arrives, but a timeout duration can be specified.
```

Comment:
That's what I get for opening this PR in my usual editor instead of just in the web interface. Do I need to revert this?

Reply:
Totally fine, doesn't really distract from this diff, and we aren't uptight about pre-commit hooks for no-trailing-space stuff.

Reply:
Cool, I wouldn't have even bothered asking if it was a full file's worth of trailing spaces that had been removed.